### PR TITLE
Connect webextension followup

### PR DESCRIPTION
**Pull request by mroz22**
couple of changes, followup for https://github.com/trezor/trezor-suite/pull/9916

- updated docs
- removed unused code from update webextensions script
- changed logger module to allow better flexibility and extandability. 
- change postMessage to accept `{useQueue: bool,  usePromise}` param
- in connect-popup, log messages are handled by only one event handler now. previously there were 2 which was causing some troubles.

### CODE REVIEW

File: packages/connect-popup/src/index.tsx [hunk 1]
```diff
@@ -513,6 +513,7 @@ const fail = (error: ErrorViewProps) => {
 
 addWindowEventListener('load', onLoad, false);
 addWindowEventListener('message', handleInitMessage, false);
+addWindowEventListener('message', handleLogMessage, false);
```
Comment (karliatto) [inline packages/connect-popup/src/index.tsx:516]: I like this idea of separating the handle messages in different functions! :+1: 
Better than all in one huge function.

File: packages/connect-common/src/messageChannel/abstract.ts [hunk 2]
```diff
@@ -173,7 +178,9 @@ export abstract class AbstractMessageChannel<
         try {
             this.sendFn(message);
         } catch (err) {
-            this.messagesQueue.push(message);
+            if (useQueue) {
+                this.messagesQueue.push(message);
```
Comment (karliatto) [inline packages/connect-common/src/messageChannel/abstract.ts:182]: Ok, so when `useQueue === true` until there is a `handshake` event that will send those messages in the queue?

**Review by karliatto — APPROVED** [review]

File: packages/connect-popup/src/index.tsx (hunk 1 above)
Comment (mroz22) [reply packages/connect-popup/src/index.tsx:516]: me too, we can now also remove specific listeners during the lifecycle of app.

File: packages/connect-common/src/messageChannel/abstract.ts (hunk 2 above)
Comment (mroz22) [reply packages/connect-common/src/messageChannel/abstract.ts:182]: reason for this `useQueue` param is that we actually don't want to queue and repeat messages during handshake, or in general, any messages that are actually expected to fail.

**Pull request merged**